portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+  private UserService userService = UserServiceFactory.getUserService();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
```

Comment:
In general we should be sending JSON unless you are sending explicit html to the client. I see your js fetch reads it as json (i wonder if some magic is happening there to make it work lol) so this probably a small oversight.

Reply:
Done. 

Seems like `response.json()` doesn't check content type. I was sending a single boolean value, so `Boolean.toString()` and `Gson.toJson()` probably gave the same thing.